lms/urls.py

```diff
@@ -10,7 +10,8 @@
 if settings.DEBUG or settings.FEATURES.get('ENABLE_DJANGO_ADMIN_SITE'):
     admin.autodiscover()
 
-urlpatterns = ('',  # nopep8
+urlpatterns = (
+    '',  # nopep8
```

Comment:
Normally, you've absolutely correct that we want to put all arguments to a function call on subsequent lines when they're too long to all put on one line. However, `urlpatterns` is a special case: because the first argument means something very different from the other arguments, the normal Django pattern is to keep the first argument on the first line with the function call. (It's structured that way in all the Django documentation.) Please revert this change back to the way it was. If pylint or pep8 is complaining, talk to me -- we can look into finding a way to silence the quality checker for this specific situation. Remember, the goal is not to blindly follow the warnings and errors that pylint/pep8 give you, the goal is to improve the readability of the code.